code/xcvm/cosmwasm/contracts/asset-registry/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
```

Comment:
I believe we need to update the registry to handle native assets as well, currently we only handle CW20 assets right? Would be nice to move to a `enum AssetReference { Native(Denom), Virtual(Address) }`